README.md

```diff
@@ -1,6 +1,6 @@
 # Terraform Cloud Operator for Kubernetes
 
-> This repository contains experimental software which is still under development and in the alpha testing stage. It is not ready for production use.
```

Comment:
Not sure how these changes came in with mine. I updated my branch with my upstream master to pull in recent changes and then I did a squash & rebase on my commits to clean them up. I can recreate my branch without these if needed.